web/themes/custom/badcamp2017/badcamp2017.theme

```diff
@@ -361,5 +360,55 @@ function badcamp2017_preprocess_flag__add_to_schedule(&$vars) {
   $vars['attributes']['class'][] = 'fa';
   $vars['attributes']['class'][] = 'button';
   $vars['attributes']['class'][] = 'medium';
+}
+
+function badcamp2017_preprocess_form_element__session_track(&$vars) {
+  $term_id = $vars['element']['#return_value'];
+  $term = \Drupal::entityTypeManager()->getStorage('taxonomy_term')->load($term_id);
+  $icon = $term->get('field_icon');
+  if (!empty($icon_class = $icon->getValue())){
+    $vars['label']['#attributes']['data_icon'] = $icon_class[0]['value'];
+  }
+}
+
+/**
+ * Implements theme_views_view__HOOK().
+ */
+function badcamp2017_preprocess_views_view_unformatted__my_schedule__full_schedule(&$vars) {
+  $title = trim(strip_tags($vars['title']->__toString()));
+  $class = '';
+  $add_title = '';
+  switch($title){
+    case 'Wednesday, October 18':
+      $add_title = t('Drupal Training & Summits');
+      $class = 'item-wed';
+      break;
+    case 'Thursday, October 19':
+      $add_title = t('Drupal Training & Summits');
+      $class = 'item-thurs';
+      break;
+    case 'Friday, October 20':
+      $add_title = t('Sessions');
+      $class = 'item-fri';
+      break;
+    case 'Saturday, October 21':
+      $add_title = t('Sessions');
+      $class = 'item-sat';
+      break;
+  }
+  $vars['add_title'] = $add_title;
+  $vars['title_class'] = $class;
+}
 
+/**
+ * Implements template_preprocess_field__HOOK().
+ */
+function badcamp2017_preprocess_field__node__field_session_track(&$vars){
+  $node = $vars['element']['#object'];
+
+  foreach($node->get('field_session_track') AS $id => $term) {
+    $full_term = $term->get('entity')->getTarget()->getValue();
+    $class = $full_term->get('field_icon')->first()->getValue()['value'];
```

Comment:
This is throwing a fatal error on the dev site: 

Error: Call to a member function      
                                          getValue() on null in                 
                                          badcamp2017_preprocess_field__node__f 
                                          ield_session_track() (line 411...

I guess we need to check for existence first?